simple_diarizer/diarizer.py

```diff
@@ -25,12 +25,16 @@ def __init__(
         assert cluster_method in [
             "ahc",
             "sc",
-        ], "Only ahc and sc in the supported clustering options"
+            "NME-sc",
+        ], "Only ahc,sc and NME-sc in the supported clustering options"
 
         if cluster_method == "ahc":
             self.cluster = cluster_AHC
         if cluster_method == "sc":
             self.cluster = cluster_SC
+        if cluster_method == "NME-sc":
```

Comment:
shouldn't it be lower cased "nme-sc" given that all other things ("ahc", "sc", ...) are also lower cased acronyms?

Reply:
done